src/Idris/Package.idr

```diff
@@ -843,6 +843,20 @@ clean pkg opts -- `opts` is not used but might be in the future
          deleteFolder builddir []
          maybe (pure ()) (\e => delete (outputdir </> e))
                (executable pkg)
+         -- clean out the generated docs
+         let build = build_dir (dirs (options defs))
+         let docBase = build </> "docs"
+         let docDir = docBase </> "docs"
+         () <- do Right docfiles' <- coreLift $ listDir docDir
```

Comment:
Looks great! Can we refactor the two do-blocks in their own auxiliary function?

Reply:
Thank you!  That sounds good, should be resolved by commit ccff608e1970f2b205b3e8ea7afdb967eb85c9dc.

Reply:
Sorry that's not quite what I meant, the two functions you put in auxiliary definitions are essentially the same, except for the file path, it would be nice to remove this redundancy.

Reply:
Ah I see!  I think I've addressed this with d788e07c327b9bf57d154d7b75304e9481c1153f.

Reply:
Not sure why [this test](https://github.com/idris-lang/Idris2/actions/runs/10199548283/job/28217862470?pr=3353) is failing ...